feat: add page on ratelimiting for Flashbots Protect (#217)
flashbots/docs: docs/flashbots-protect/rpc/ratelimiting.md

---
title: Rate limiting
---

> In order to protect our service from abuse, we have rate limits on the number of requests that can be made to Flashbots Protect RPC. In particular you can only make 80 rpc requests per second with a burst limit of up to 100 requests. Note that this is *requests* per second and not *transactions* submitted per second. If you are making more than 80 requests per second on average your requests will be blocked for 1 minute.

Reviewer: ```suggestion
We have rate limits on the number of requests that can be made to Flashbots Protect RPC in order to protect our service from abuse. Currently you can only make 80 requests per second with a burst limit of up to 100 requests. If you are making more than 80 requests per second on average your requests will be blocked for 1 minute.

Note that this is *requests* per second and not *transactions* submitted per second. The number of transactions in a request is not limited. 
```

> Note that you are not required to read json rpc requests to send transactions to Flashbots Protect RPC.

Reviewer: ```suggestion
Note that you are not required to read JSON RPC requests to send transactions to Flashbots Protect RPC.
```

Reviewer: I don't understand this part. Why are you including it?

Reviewer 2: Agree, I'd remove it too.

---
title: Rate limiting
---

> We have rate limits on the number of requests that can be made to Flashbots Protect RPC in order to protect our service from abuse. Currently you can only make 80 requests per second with a burst limit of up to 100 requests. If you are making more than 80 requests per second on average your requests will be blocked for 1 minute.

Reviewer 2: ```suggestion
We have rate limits on the number of requests that can be made to Flashbots Protect RPC in order to protect our service from abuse. Currently you can only make 80 requests per second with a burst limit of up to 100 requests.
```

Reviewer 2: A user is not blocked for a minute, just for a second.
